tools/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0
 Since this project is only a vessel for packaging a few binary tools, its adherence to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) is loose at best.
 
+## [0.2.0] - 2022-09-20
+
+Update eksctl and sonobuoy.
+
+### Contents
+
+- boringtun v0.4.0
```

Comment:
I thought I had already removed this.

Reply:
I'll check. The change log hadn't been changed since it was added so I just changed the eksctl bit.

Reply:
Borington is removed. I'll fix the log